### PR TITLE
Add this feature i made

### PR DESCRIPTION
**Pull request by ElPro7777**
Add this feature i made Sunky scratch

### CODE REVIEW

**Review by rgantzos — CHANGES_REQUESTED** [review]
We don't need an entire extra file for the code, it won't do anything (I'm referring to Sunny-scratch.js

**Comment by ElPro7777** [comment]
its my first try to use Javascript idk what i made

**Comment by rgantzos** [comment]
@ElPro7777 did you do any testing?

**Comment by rgantzos** [comment]
It freezes 2.0 pages, but it won’t freeze them if you put the CSS on the main page element, not the body.

File: Sunky-scratch.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+alert(Sunky Says: Hello)
```
Comment (veloce-vasta) [inline Sunky-scratch.js:1]: You forgot ""

File: Sunky-scratch.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+.rotate90 {
```
Comment (veloce-vasta) [inline Sunky-scratch.js:2]: This is not how you insert css with js

**Comment by callumjt** [comment]
loads of issues with the code and really old so it probably wont be fixed, closing

**Pull request closed without merging**